File: DataStructure/C#/Employee.cs
class Employee
{
    public Personal personal    = new Personal();
    public Salary   salary      = new Salary();

    public class Personal
    {
        public string name = "";
        public int age = 0;
    }

    public class Salary
    {
        public bool raise = false;
        public TaxCode taxCode = TaxCode.LOWINCOME;
    }
}

File: DataStructure/Java/src/Employee.java
public class Employee {
    public Personal personal    = new Personal();
    public Salary   salary      = new Salary();

    public class Personal {
        public String name = "";
        public int age = 0;
    }

    public class Salary {
        public boolean raise = false;
        public TaxCode taxCode = TaxCode.LOWINCOME;
    }
}